File: isoft_android/linkknown/app/src/main/java/com/linkknown/ilearning/model/Paginator.java
package com.linkknown.ilearning.model;

import java.util.List;

import lombok.Data;

@Data
public class Paginator {

    private int currpage;
    private int firstpage;
    private int lastpage;
    private List<Integer> pages;
    private int pagesize;
    private int totalcount;
    private int totalpages;
}

File: isoft_flutter/linkknown/lib/model/paginator.dart
import 'package:json_annotation/json_annotation.dart';

@JsonSerializable()
class Paginator {
  int currpage;
  int firstpage;
  int lastpage;
  List<int> pages;
  int pagesize;
  int totalcount;
  int totalpages;

  Paginator(this.currpage, this.firstpage, this.lastpage, this.pages,
      this.pagesize, this.totalcount, this.totalpages);

}
